config/config/sc-config.yaml

```diff
@@ -114,11 +114,11 @@ grafana:
     sidecar:
       resources:
         requests:
-          cpu: 10m
-          memory: 80Mi
+          cpu: 80m
+          memory: 640Mi
```

Comment:
this is a huge increase in memory requests. is this needed?

Reply:
Yep, for sure was a bit questionable at first. Have tried to narrow it down now.